public/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "airbnb-base",
```

Comment:
Just for future reference...  ESLint has a [cascading and hierarchical model](https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) for configurations.  So, if in the future, you want to share a more complicated configuration between back-end and front-end code, it should Just Work in the current project layout.

**TL;DR:** This line shouldn't be necessary as it should be picked up from the parent configuration (unless there's something special about the `extends` property I don't know about).